.github/actions/run-tests/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM golang:1.15-alpine
 
 # Add any build or testing essential system packages
 RUN apk add --no-cache build-base git
+RUN apk add gcc pkgconfig libc-dev
```

Comment:
`gcc` and `libc-dev` are part of `build-base`. You can add `pkgconf` (`pkgconfig` is an alias) on the previous line.